packages/cache/src/prefixer.ts

```diff
@@ -49,87 +19,12 @@ function prefix(value: string, length: number): string {
     case 6135:
     case 4599:
     case 4855:
-    // background-clip, columns, column-(count|fill|gap|rule|rule-color|rule-style|rule-width|span|width)
-    case 4215:
-    case 6389:
-    case 5109:
-    case 5365:
-    case 5621:
-    case 3829:
       return WEBKIT + value + value
-    // appearance, user-select, transform, hyphens, text-size-adjust
-    case 5349:
+    // user-select, hyphens, text-size-adjust
```

Comment:
- user-select still prefixed in webkit https://caniuse.com/mdn-css_properties_user-select, moz prefix has been unnecessary since 2019
- hyphens will be widely available early 2026 https://caniuse.com/css-hyphens, moz prefix has been unnecessary since 2015
- text-size-adjust still prefixed in mobile safari https://caniuse.com/text-size-adjust, seemingly never implemented in firefox so unsure why moz prefix was here 

For all of these, we don't need the MS prefix since that's all IE/pre-Chromium Edge which are old enough to not matter for "Widely Available".

So none of these need `moz`/or `ms` prefixes